tests/CMakeLists.txt

```diff
@@ -133,6 +133,11 @@ endif()
 
 if (MSVC)
 	add_compile_options(/MP) # Enable multi parrallel build
+	if (MSVC_VERSION LESS_EQUAL 1900)
```

Comment:
I don't see the reason why appveyor doesn't pick up this condition and does not disable those specific warnings? [See appveyor build](https://ci.appveyor.com/project/fnc12/sqlite-orm/builds/43393529/job/r9kh99ga09hrqhm8).

Reply:
Actually I am not a master in CMake and CI configs but let me try to read it and understand how it works